File: python/1221.py
from math import sqrt

n = int(input())

for i in range(n):
    x = int(input())
    sqrtX = int(sqrt(x))
    
    flag = False
    
    for j in range(2, sqrtX + 1):
        if x % j == 0:
            flag = True
            break 
        
    if flag == True:
        print('Not Prime')   
    else:
        print('Prime')

File: lua/1221.lua
local x = io.read('n')
for j = 1, x do
    local n = io.read('n')
    local flag = false
    local nSqrt = tonumber(math.sqrt(n))
    for i = 2, nSqrt do
        if n % i == 0 then
            print('Not Prime')
            flag = true
            break
        end
    end
    if not flag then
        print('Prime')
    end
end